src/CommunityToolkit.Maui/Views/AvatarView.cs

```diff
@@ -0,0 +1,333 @@
+using System.ComponentModel;
```

Comment:
The file name should be AvatarView.shared.cs

Reply:
Renamed.